source/utilities.tex

```diff
@@ -1601,10 +1599,7 @@
 
 \begin{itemdescr}
 \pnum
-The pack \tcode{VTypes} is defined as follows. Let \tcode{U}$_i$ be \tcode{decay_t<T$_i$>} for each
-\tcode{T}$_i$ in \tcode{TTypes}. If \tcode{U}$_i$ is a specialization of
-\tcode{reference_wrapper}, then \tcode{V}$_i$ in \tcode{VTypes} is \tcode{U$_i$::type\&},
-otherwise \tcode{V}$_i$ is \tcode{U}$_i$.
+Let \tcode{VTypes...} be \tcode{unwrap_ref_decay_t<TTypes>...}.
 
 \pnum
 \returns \tcode{tuple<VTypes...>(std::forward<TTypes>(t)...)}.
```

Comment:
Similarly, the "detour" through the definition of `VTypes...` here only complicates this specification.

Reply:
Done.